tutorial.md

```diff
@@ -307,13 +307,21 @@ First we need to setup a Pod sandbox using a Pod configuration, which can be fou
 cd $GOPATH/src/github.com/kubernetes-incubator/cri-o
 ```
 
+
+In case the file /etc/containers/policy.json does not exist on your filesystem, copy it from the source tree:
+
+```
+sudo mkdir /etc/containers/
+sudo cp test/policy.json /etc/containers
```

Comment:
This is OK for a test system, but production wise I don't think you want to use this particular policy.json.  The better answer is to be sure that 'skopeo-constainers' was installed as noted at line 282.

Reply:
Yeah the policy seems to be a "bit insecure", but since we're executing all this on a throwaway VM for tutorial purposes it didn't look wrong. I can revert that back though. Once Ubuntu 18.04 .debs will be available for skopeo that will likely be useless anyway.